recipe/meta.yaml

```diff
@@ -1,30 +1,22 @@
-{% set version = "0.7.0" %}
-{% set hash = "43e6a9fcb146ad871515f0d0873947e5d497a1c9c60c58cb102a97b47208b7c3" %}
-{% set build_number = "1" %}
+{% set version = "0.8.0" %}
 
```

Comment:
```suggestion
{% set name = "yaml-cpp" %}
```

Reply:
As is usual with me, I would like to not diverse too much from conda-forge if possible.